sdk/go/x/auto/stack.go

```diff
@@ -924,3 +989,25 @@ func (s *languageRuntimeServer) GetPluginInfo(ctx context.Context, req *pbempty.
 		Version: "1.0.0",
 	}, nil
 }
+
+func tailLogs(command string, receivers []chan<- events.EngineEvent) (*tail.Tail, error) {
+	logDir, err := ioutil.TempDir(os.TempDir(), fmt.Sprintf("automation-logs-%s-", command))
```

Comment:
Do you think we should clean up the temp file on behalf of the user? Looks like we could make that addition to the cleanup function before without too much trouble. 

We are opting them into this behavior on preview since we rely on it for even summary counts. Just in terms of limiting surprises.

Reply:
Yeah, good call. Should do for nodejs as well. Also learned that if you just pass in an empty string as the first arg to `ioutil.TempDir` it automatically uses `os.TempDir()`